scripts/deployment_manager/configs/darwin.json

```diff
@@ -0,0 +1,106 @@
+{
+  "version": "0.2",
+  "components": {
+    "setupvars": {
+      "mandatory" : "yes",
+      "files": [
+        "bin"
+      ]
+    },
+    "openvino_license": {
+      "mandatory" : "yes",
+      "files": [
+        "licensing"
+      ]
+    },
+    "ie_core": {
+      "group": ["ie"],
+      "files": [
+        "deployment_tools/inference_engine/version.txt",
+        "deployment_tools/inference_engine/lib/intel64/libinference_engine.dylib",
+        "deployment_tools/inference_engine/lib/intel64/libinference_engine_lp_transformations.dylib",
+        "deployment_tools/inference_engine/lib/intel64/libinference_engine_transformations.dylib",
+        "deployment_tools/inference_engine/lib/intel64/libinference_engine_preproc.so",
+        "deployment_tools/inference_engine/lib/intel64/libinference_engine_ir_reader.so",
+        "deployment_tools/inference_engine/lib/intel64/libinference_engine_onnx_reader.so",
+        "deployment_tools/inference_engine/lib/intel64/libinference_engine_c_api.dylib",
+        "deployment_tools/inference_engine/lib/intel64/libAutoPlugin.so",
```

Comment:
as far as I remember, we also agreed to add HETERO and MULTI plugins to core part

Reply:
Nope, we don't have such agreement - they can be added as separated plugins.
Let's migrate code AS IS and then start working on feature requests